src/Users/User.php

```diff
@@ -18,19 +18,24 @@ class User implements UserInterface
     /** @var string */
     protected $username;
 
+    /** @var array */
+    protected $user_info;
+
     /**
      * User constructor.
      * @param $id
      * @param $first_name
      * @param $last_name
      * @param $username
+     * @param $user_info
      */
-    public function __construct($id = null, $first_name = null, $last_name = null, $username = null)
+    public function __construct($id = null, $first_name = null, $last_name = null, $username = null, $user_info = [])
```

Comment:
In fact $user_info may be an object, this is why I do not typehint it.
But we cast $user_info after as an array, which is not the same.